docs/reference/src/main/asciidoc/environments.asciidoc

```diff
@@ -723,6 +723,14 @@ Another approach is to use `Weld.property()` and `SeContainerInitializer.addProp
 
 NOTE: The bean discovery mode of `annotated` is supported from version 2.2.0.Final. Previous versions processed implicit bean archives in the same way as explicit bean archives.
 
+==== Extending Bean Defining Annotations
+
+If you are running with discovery mode `annotated`, then only classes with bean defining annotations will be picked up as beans.
+The set of these annotations is given by CDI but Weld SE allows you to expand it via `Weld.addBeanDefiningAnnotations(Class<? extends Annotation>... annotations)`.
+Any annotation added this via will be considered a bean defining annotation when performing discovery.
```

Comment:
Look like a wrong word order: this via -> via this? But I don't like "via this" too much, maybe e.g. "this way" would be better?

Reply:
Sure, that's a typo, meant to be `this way`, thanks.